src/util/WorkflowsExplorer/Attempt.ts

```diff
@@ -0,0 +1,85 @@
+import { ActionsState, Run, RunInfo, StateFile, Row } from "../../types"
+import randomStateFile from "./randomStateFile";
+
+
+export default class Attempt {
+    runInfo: RunInfo; // Contains all general information
+    rows: Row[];
+    run: Run;
+    
+    constructor(stateFile?: StateFile) {  
+        if (stateFile) {
+            this.runInfo = this.getRunInfo(stateFile);  
+            this.rows = this.getTaskRow(stateFile.actionsState).sort(this.cmp);
+            this.run =  this.getRun();
+        } else {
+            const stateFile = randomStateFile();
```

Comment:
This is just used for testing purposes, right?

Reply:
Exactly